### ISSUE TITLE
the basics

### ISSUE THREAD

**Issue by zwazoe** [hexojs/warehouse#43]
at least show exammples of the very basic CRUD. 

People have to climb mountain to find how to Read all

**Comment by tcrowe** [comment]
Agreed. This documentation could be improved with examples and more help. Thank you, @zwazoe
